util/util.go

```diff
@@ -58,3 +63,50 @@ func AddImageNames(store storage.Store, image *storage.Image, addNames []string)
 	}
 	return nil
 }
+
```

Comment:
Probably better to move this to `cmd/buildah/common.go`, since we don't use it outside of the `cmd` package.